code/datums/datumvars.dm

```diff
@@ -995,6 +995,39 @@
 			manipulate_organs(C)
 			href_list["datumrefresh"] = href_list["editorgans"]
 
+		else if(href_list["brainwash"])

```

Comment:
I mean using admin_add in antag datum, i guess i should have been more clear.

Reply:
Oh, i didn't know there was a proc for that, will fix